versions/index.html

```diff
@@ -454,6 +454,8 @@
+,{"version":"24.12.1.1614","actual_version":"24.12.1.1614","result":[[0.04,0.009,0.009,0.071,0.008],[0.02,0.01,0.011,0.009,0.01],[0.054,0.019,0.012,0.04,0.012],[0.027,0.011,0.011,0.028,0.011],[0.039,0.023,0.024,0.046,0.022],[0.144,0.019,0.022,0.04,0.018],[0.023,0.006,0.006,0.005,0.006],[0.036,0.018,0.017,0.019,0.018],[0.069,0.045,0.045,0.065,0.044],[0.062,0.038,0.035,0.034,0.036],[0.277,0.243,0.241,0.237,0.271],[0.453,0.275,0.274,0.311,0.263],[0.037,0.003,0.003,0.003,0.003],[0.016,0.009,0.009,0.009,0.008],[0.413,0.309,0.301,0.323,0.263],[0.235,0.063,0.062,0.06,0.06],[0.172,0.085,0.072,0.072,0.071],[0.138,0.084,0.067,0.066,0.065],[0.063,0.03,0.033,0.057,0.032],[0.181,0.133,0.135,0.133,0.131],[0.16,0.107,0.106,0.103,0.103],[0.162,0.125,0.121,0.125,0.122],[1.167,0.568,0.557,0.553,0.546],[1.556,0.581,0.58,0.597,0.582],[0.148,0.098,0.101,0.09,0.085],[0.103,0.078,0.077,0.102,0.072],[0.066,0.047,0.041,0.042,0.044],[0.074,0.041,0.04,0.039,0.038],[0.002,0.003,0.003,0.002,0.002],[0.023,0.009,0.009,0.01,0.009],[0.043,0.02,0.019,0.042,0.02],[0.107,0.026,0.025,0.025,0.025],[0.149,0.083,0.083,0.083,0.084],[0.27,0.201,0.188,0.187,0.183],[0.003,0.003,0.003,0.026,0.003],[0.014,0.012,0.012,0.012,0.012],[0.322,0.252,0.257,0.252,0.251],[0.361,0.292,0.29,0.301,0.283],[0.179,0.095,0.095,0.093,0.11],[0.187,0.112,0.13,0.131,0.111],[0.474,0.349,0.34,0.333,0.336],[0.591,0.452,0.452,0.436,0.438],[0.51,0.405,0.392,0.381,0.379],[0.41,0.282,0.277,0.276,0.276],[1.27,0.941,0.934,0.934,0.943],[0.72,0.597,0.6,0.564,0.56],[2.081,1.816,1.795,1.773,1.802],[0.105,0.026,0.026,0.025,0.025],[2.777,0.483,0.487,0.484,0.505],[3.319,0.604,0.583,0.601,0.59],[6.706,1.381,1.384,1.425,1.396],[4.161,0.652,0.619,0.624,0.608],[0.347,0.154,0.152,0.151,0.146],[0.199,0.119,0.118,0.116,0.121],[0.345,0.153,0.146,0.149,0.148],[2.792,0.556,0.564,0.577,0.543],[2.239,0.767,0.772,0.781,0.77],[0.055,0.035,0.035,0.056,0.036],[0.481,0.304,0.295,0.299,0.281],[1.26,0.385,0.38,0.377,0.379],[2.614,1.809,1.778,1.818,1.833],[3.378,1.726,1.679,1.69,1.689],[3.419,1.699,1.684,1.673,1.677],[0.343,0.213,0.214,0.214,0.219],[0.067,0.038,0.04,0.036,0.04],[0.043,0.031,0.032,0.032,0.029],[0.046,0.027,0.026,0.022,0.02],[0.105,0.073,0.071,0.063,0.065],[0.017,0.009,0.009,0.009,0.009],[0.017,0.009,0.009,0.009,0.009],[0.009,0.005,0.005,0.005,0.005],[0.442,0.358,0.366,0.366,0.355],[15.215,1.434,1.41,1.476,1.4],[2.169,1.569,1.541,1.54,1.542],[18.392,4.788,4.798,4.772,4.778]],"source":"results/24.12.1.1614.json"}
```

Comment:
No need to update index.html manually, it is updated by a CI job after the PR is merged.